code/game/objects/items.dm

```diff
@@ -755,3 +764,24 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 	SEND_SIGNAL(src, COMSIG_ITEM_ATTACK_SELF, user)
 	if(interaction_flags_item & INTERACT_ITEM_ATTACK_SELF)
 		interact(user)
+
+//? Mob Armor
+
+/**
+ * called to be checked for mob armor
+ * 
+ * @returns copy of args with modified values
+ */
+/obj/item/proc/checking_mob_armor(damage, tier, flag, mode, attack_type, datum/weapon, target_zone)
+	damage = fetch_armor().resultant_damage(damage, tier, flag)
+	return args.Copy()
+
+/**
+ * called to be used as mob armor
+ * side effects are allowed
+ * 
+ * @returns copy of args with modified values
+ */
+/obj/item/proc/running_mob_armor(damage, tier, flag, mode, attack_type, datum/weapon, target_zone)
```

Comment:
isnt this the exact same result as above when called?

Reply:
check = no side effects, running = side effects

in rigsuit update absorbed damage goes directly to the armor, but if we're just checking armor we don't want it to take damage